src/tools/cephfs/cephfs-shell

```diff
@@ -773,6 +773,129 @@ class CephFSShell(Cmd):
 
         return 0
 
+    def complete_ln(self, text, line, begidx, endidx):
+        """
+        auto complete of file name.
+        """
+        return self.complete_filenames(text, line, begidx, endidx)
+
+    ln_parser = argparse.ArgumentParser(
+        description='Create a hard link or a symbolic '
+                    'link to an existing file or directory.')
+    ln_parser.add_argument('source_path', type=str, action=path_to_bytes,
+                           help='Path of the source file')
+    ln_parser.add_argument('destination_path', type=str, action=path_to_bytes,
```

Comment:
nit: opinion: same, would be easier if `destination_path` is path `d_path`.

Reply:
what about `destination`?

Reply:
IMO better since shorter.

Reply:
Right

Reply:
Done